### PR TITLE
docs: Use SSL or Insecure clients in examples

### PR DESCRIPTION
**Pull request by adityapk00**
## Motivation

Using `getHubRpcClient` can be slow because it attempts a SSL connection first, and has to wait for a timeout. Update the docs to use `getSSLHubRpcClient` or `getInsecureHubRpcClient` directly. 

## Change Summary

- Update docs

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [X] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](../CONTRIBUTING.md#35-adding-changesets)
- [X] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [X] PR does not require changes to the [protocol](https://github.com/farcasterxyz/protocol)

### CODE REVIEW

**Review by varunsrin — COMMENTED** [review]
1. should we deprecate `getHubRpcClient` entirely? the slower performance and unclear security behavior seem like they might lead users to footguns. if we do this, we'll need to bump the minor version in the package. 

2. connecting to a non-SSL hub with ssl or vice versa returns the error `  error: HubError: No connection established`. Is it possible we can throw a more descriptive error here to help users understand that they are using the wrong authentication method?

**Comment by adityapk00** [comment]
For 1. Yes, we probably should. I'll update this PR with deprecating that. 
For 2. Unfortunately we don't have more info here, since we don't know for sure that the wrong method was used. We can't differentiate between:
a. Connecting to SSL but server is not SSL (or vice-versa)
b. Couldn't reach the server or Could reach the server, but there wasn't anything listening on the port. 

Hence the generic message. We could add a hint here, saying something like "No connection established. Are you sure you are using SSL?" or something. Does that make sense?

File: .changeset/chilly-bats-kneel.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@farcaster/hub-nodejs': patch
+'@farcaster/protobufs': patch
+'@farcaster/utils': patch
```
Comment (varunsrin) [inline .changeset/chilly-bats-kneel.md:4]: should be minor version for hub-nodejs and utils since we're "breaking" the API

Comment (adityapk00) [reply .changeset/chilly-bats-kneel.md:4]: Oh, right. Sorry, will fix

**Pull request merged**